docs/resources/grant_account_role.md

```diff
@@ -43,7 +43,7 @@ resource "snowflake_user" "user" {
 }
 
 resource "snowflake_grant_account_role" "g" {
-  role_name = snowflake_role.role.name
+  role_name = snowflake_account_role.role.name
```

Comment:
Hey @joscha 
Please make the same change in examples/resources/snowflake_grant_account_role/resource.tf. That would be sufficient to merge the pr.
If you would like to make future contributions to the documentation, please mind that it's generated. In this case, you could only modify examples/resources/snowflake_grant_account_role/resource.tf and run `make docs` to generate the .md file you changed manually here. Before running `make docs` it's necessary to run `make dev-setup` before, to download the tool responsible for doc generation.

Reply:
I see, thank you, done in [19e64d6](https://github.com/Snowflake-Labs/terraform-provider-snowflake/pull/3101/commits/19e64d64fa8b3b842452b85602962b60e8658043)